<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurCrawler</class>
 <widget class="QWidget" name="ServeurCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurCrawler</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>591</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelPortEcoute">
      <property name="text">
       <string>Port d'Ã©coute</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxPortServeur">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>8888</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLancementServeur">
      <property name="text">
       <string>Lancement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurCrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
